app/src/main/java/com/android/sample/ui/utils/TimeDisplay.kt

```diff
@@ -0,0 +1,23 @@
+package com.android.sample.ui.utils
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import com.android.sample.R
+import com.android.sample.resources.C.Tag.MINUTES_PER_HOUR
+
+/**
+ * Composable function to reformat the time.
+ *
+ * @param time The time to reformat in minutes.
+ */
+@Composable
+fun reformatTime(time: Float): String {
```

Comment:
- **Code Organization**: Moving `reformatTime` into a dedicated file is a good decision. It keeps utility functions centralized and reusable. 👍

- **Clarity**: The function is clear and well-documented. The use of constants (`MINUTES_PER_HOUR`) and `stringResource` makes the code readable and adaptable for localization. 👌

 **Suggestion**: Since this is a utility, you might consider writing a unit test to validate the formatting for edge cases (e.g., 0 minutes, exactly 60 minutes, and very large values).

Reply:
Thanks for the comment, will do it right away !